crates/sui-framework/sources/epoch_time_lock.move

```diff
@@ -5,9 +5,9 @@ module sui::epoch_time_lock {
     use sui::tx_context::{Self, TxContext};
 
     /// The epoch passed into the creation of a lock has already passed.
-    const EEPOCH_ALREADY_PASSED: u64 = 0;
+    const EEpochAlreadyPassed: u64 = 0;
     /// Attempt is made to unlock a lock that cannot be unlocked yet.
-    const EEPOCH_STILL_LOCKED: u64 = 1;
+    const EEpochStilLocked: u64 = 1;
```

Comment:
```suggestion
    const EEpochStillLocked: u64 = 1;
```

Reply:
Although: as long as we're renaming this, I think something like `EEpochOngoing` or `EEpochNotYetEnded` might be more intuitive--saying that the "epoch is locked" seems a bit confusing (especially since the "already passed" error message doesn't mention a lock.